### PR TITLE
Display parent account info above account name

### PR DESCRIPTION
**Pull request by ekowalsk**
image
image
image

### CODE REVIEW

**Comment by krzysztof-jelski** [comment]
Styling remark: in the designs the account name is aligned all the time with middle of identicon. Would be good if we could preserve that.
image

File: packages/page-accounts/src/Accounts/Account.tsx [hunk 1]
```diff
@@ -472,7 +491,11 @@ function Account ({ account: { address, meta }, className = '', delegation, filt
           )}
         </td>
         <td className='address'>
-          <AddressSmall value={address} />
+          <AddressSmall
+            parentAddress={(typeof meta.parentAddress === 'string') ? meta.parentAddress : undefined}
```
Comment (jacogr) [inline packages/page-accounts/src/Accounts/Account.tsx:495]:
```suggestion
            parentAddress={meta.parentAddress || undefined}
```

But even the `|| undefined` should not be needed, these should just be string or undefined (if null, should also be all ok)

File: packages/react-components/src/AddressSmall.tsx [hunk 2]
```diff
@@ -6,6 +6,9 @@ import type { AccountId, Address } from '@polkadot/types/interfaces';
 import React from 'react';
 import styled from 'styled-components';
 
+import ParentAddress from '@polkadot/react-components/ParentAddress';
+import { toShortAddress } from '@polkadot/react-components/util';
```
Comment (jacogr) [inline packages/react-components/src/AddressSmall.tsx:10]:
```suggestion
import ParentAddress from './ParentAddress';
import { toShortAddress } from './util';
```

We are in the same package.

File: packages/react-components/src/ParentAddress.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+      </AccountName>
+    </div>
+  );
+}
+
+export default React.memo(styled(ParentAddress)`
+    display: flex;
+    align-items: center;
+    color: #8B8B8B;
+    font-size: 0.75rem;
```
Comment (jacogr) [inline packages/react-components/src/ParentAddress.tsx:35]:
```suggestion
  align-items: center;
  color: #8B8B8B;
  display: flex;
  font-size: 0.75rem;
```

**Review by jacogr — COMMENTED** [review]
Love the display.

File: packages/page-accounts/src/Accounts/Account.tsx (hunk 1 above)
Comment (ekowalsk) [reply packages/page-accounts/src/Accounts/Account.tsx]: ✅

File: packages/react-components/src/AddressSmall.tsx (hunk 2 above)
Comment (ekowalsk) [reply packages/react-components/src/AddressSmall.tsx]: ✅

File: packages/react-components/src/ParentAddress.tsx (hunk 3 above)
Comment (ekowalsk) [reply packages/react-components/src/ParentAddress.tsx]: ✅

**Review by jacogr — APPROVED** [review]
Thank you.

**Pull request merged**